app/tests/tap-dance/behavior_keymap.dtsi

```diff
@@ -0,0 +1,45 @@
+#include <dt-bindings/zmk/keys.h>
+#include <behaviors.dtsi>
+#include <dt-bindings/zmk/kscan_mock.h>
+
+/ {
+	behaviors {
+
+        tdb: tap_dance_basic {
+            compatible = "zmk,behavior-tap-dance";
+            label = "TAP_DANCE_BASIC";
+            #binding-cells = <0>;
+            tapping-term-ms = <200>;
+            bindings = <&kp N1>, <&kp N2>, <&kp N3>;
+        };
+
+        ht: hold_tap {
+            compatible = "zmk,behavior-hold-tap";
+            label = "HOLD_TAP";
+            #binding-cells = <2>;
+            tapping-term-ms = <200>;
+            quick_tap_ms = <0>;
+            flavor = "tap-preferred";
+            bindings = <&kp>, <&kp>;
+        };
+
+        tdm: tap_dance_mod {
+            compatible = "zmk,behavior-tap-dance";
+            label = "TAP_DANCE_MOD";
+            #binding-cells = <0>;
+            tapping-term-ms = <200>;
+            bindings = <&ht LSHIFT A>, <&ht LALT B>, <&ht LGUI C>;
+        };
+    };
+
+	keymap {
+		compatible = "zmk,keymap";
+		label ="Default keymap";
+
+		default_layer {
+			bindings = <
+				&tdb   &kp N0
```

Comment:
`&tdb` is not exercised in any test?

Reply:
`&tdb` is not used in testing since `&tdm` gives a better indication of held keys working properly with tap-dances. Further updates on my local repo remove `&tdb` and sets `&tdm` to the `row 0` `col 0` position (position 0) in the keymap.

Reply:
I think you need to have multiple tapdances in the keymap to prove there are no bugs when two are used shortly after each other.